packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/ios/initializeflow.md

```diff
@@ -0,0 +1,45 @@
+You can provide the configuration information in a property list file or in the initialization call.
+
+#### Add configuration with a property list
+
+The `init()` function of `InteractionCodeFlow` looks for a property list called `Okta.plist`. You can also call `init(plist:)` to specify a file.
```

Comment:
This para reads weirdly when immediately followed by the first sentence of the next paragraph. It repeats the same information enough to be jarring.

Reply:
Good catch, especially because that last sentence is no longer correct. I replaced the "You can also..." sentence with this:

> You can also call `init(issuer:clientId:scopes:redirectUri:additionalParameters:)` to set the configuration values.